src/pymortests/demos.py

```diff
@@ -44,6 +44,7 @@
     ('burgers', ['--num-flux=lax_friedrichs', '0.1']),
     ('burgers', ['--num-flux=engquist_osher', '0.1']),
     ('burgers', ['--num-flux=simplified_engquist_osher', '0.1']),
+    ('linear_optimization', [40, 20]),
```

Comment:
Can you comment on the runtime impact in CI? Relative to other demos?

Reply:
The runtime is about 2 seconds atm. I reduced this quite a bit.